src/store/index.js

```diff
@@ -41,14 +38,13 @@ export default new Vuex.Store({
     nodeStatus: NODE_STATUS_CONNECTING,
     notificationSettings: [],
     chainNames: null,
-    tip: null,
     migrations: {},
     backedUpSeed: null,
     saveErrorLog: true,
     loginTargetLocation: { name: 'account' },
-    cardMinified: false,
     qrScannerOpen: false,
     hiddenCards: [],
+    lastRoute: null,
```

Comment:
I strongly suggest to not add any new stuff to the store. Maybe the UI (`useUi`) composable would fit the purpose of holding the `lastRoute`?